lib/imgbundler/imgbundler.go

```diff
@@ -0,0 +1,136 @@
+package imgbundler
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+
+	"oss.terrastruct.com/d2/lib/xmain"
+)
+
+var imageRe = regexp.MustCompile(`<image href="([^"]+)"`)
```

Comment:
I think it's time to parse the svg correctly. I can see someone embedding some HTML code and us screwing it up.

Reply:
i'd prefer to delay for another minor version or two. the SVG structure is still changing rapidly, as bugs are being fixed in the render and functionality is added (e.g. latex). it should stabilize much more after this batch of changes.

Reply:
https://github.com/terrastruct/d2/issues/258